pkg/kubelet/cm/dra/manager.go

```diff
@@ -342,26 +343,8 @@ func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
 			continue
 		}
 
-		// Query claim object from the API server
-		resourceClaim, err := m.kubeClient.ResourceV1alpha2().ResourceClaims(pod.Namespace).Get(
-			context.TODO(),
-			*claimName,
-			metav1.GetOptions{})
-		if err != nil {
-			return fmt.Errorf("failed to fetch ResourceClaim %s referenced by pod %s: %+v", *claimName, pod.Name, err)
-		}
-
-		// Grab the allocation.resourceHandles. If there are no
-		// allocation.resourceHandles, create a single resourceHandle with no
-		// content. This will trigger processing of this claim by a single
-		// kubelet plugin whose name matches resourceClaim.Status.DriverName.
-		resourceHandles := resourceClaim.Status.Allocation.ResourceHandles
-		if len(resourceHandles) == 0 {
-			resourceHandles = make([]resourcev1alpha2.ResourceHandle, 1)
-		}
-
```

Comment:
The unit tests we introduced expect that the resource claim has been properly created and we test the case when getting the resource claim from the manager fails. I guess this needs to be updated now with your changes

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/dra/manager_test.go#L1013-L1030

Reply:
@TommyStarK Thanks for pointing out to this. I understand why "failed to fetch resource claim" test case fails, but I'm still wondering why "plugin does not exist" and "should timeout" test cases started to fail. Any ideas?

Reply:
Should be fixed now.